eng/pipelines/common/templates/pipeline-with-resources.yml

```diff
@@ -59,6 +68,12 @@ resources:
       env:
         ROOTFS_DIR: /crossrootfs/ppc64le
 
+    - container: debian-11-gcc12-amd64
+      image: mcr.microsoft.com/dotnet-buildtools/prereqs:debian-11-gcc12-amd64
+
+    - container: linux_x64_llvmaot
+      image: mcr.microsoft.com/dotnet-buildtools/prereqs:centos-7
+
```

Comment:
Why are we keeping the LLVMAOT build on the centos7 image? Same as the previous case?

Keeping this on CentOS 7 will delay some future infrastructure improvements in the repo (like upgrading CMake).

Reply:
hmm yeah we should be able to pass in the clang tools instead of binutils (I know xamarin-android does this already), can you please file a separate issue so we can take a look at that?